### PR TITLE
 Implementing CtrlMsg using struct

### PR DESCRIPTION
**Pull request by madsbk**
Implementing `CtrlMsg` using the `struct` module instead of cython's struct.

This PR is part of the effort to restrict the use of Cython to `ucx_api.pyx`

### CODE REVIEW

**Comment by madsbk** [comment]
> LGTM. There's also no change in behavior here, right?

Yes, this is only cleanup :)

**Comment by madsbk** [comment]
Yes, my plan is to make core.pyx pure python and merge it with public_api.

**Comment by jakirkham** [comment]
Sounds good. Thanks for working on this Mads 😄

File: ucp/_libs/core.pyx [hunk 1]
```diff
@@ -75,52 +76,62 @@ async def exchange_peer_info(endpoint, msg_tag, ctrl_tag, guarantee_msg_order):
...
+    """
 
-cdef class CtrlMsg:
-    cdef CtrlMsgData data
+    fmt = "QQ"
+    nbytes = struct.calcsize(fmt)
 
+    @staticmethod
+    def serialize(opcode, close_after_n_recv):
+        return struct.pack(CtrlMsg.fmt, int(opcode), int(close_after_n_recv))
```
Comment (quasiben) [inline ucp/_libs/core.pyx:92]: I know this is already merged but wanted to say thanks to @madsbk for this work.  I raised some concerns in a [previous PR](https://github.com/rapidsai/ucx-py/pull/418#discussion_r389231002) (probably wrong) and I appreciate you taking a look.

Comment (jakirkham) [reply ucp/_libs/core.pyx:92]: Yeah Mads is right that it should have been `"i"` in that context. Though it seems we have dropped that value since.

**Pull request merged**